src/agent_factory/tools.py

```diff
@@ -35,7 +48,7 @@ def search_mcp_servers(keyword: str, is_official: bool = False) -> list[dict[str
         official_servers = filter(lambda server: server.get("is_official", False), servers)
         return list(official_servers)
```

Comment:
Should we also cleanup the official servers?

Reply:
nice catch. I didn't notice that 😅 .

Will update the code and test